src/style.css

```
@@ -0,0 +1,111 @@
+body {
+  margin: 0;
+  background-color: #ccc;
+}
+
+h2 {
+  margin: 0;
+}
+
+.product_card-outer {
+  font-family: "Roboto", sans-serif;
+  font-size: 11px;
+  font-weight: 400;
+  width: 200px;
+  height: 403px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  box-sizing: border-box;
+  padding: 0 14px;
+}
+
+.product_photo {
+  display: flex;
+  justify-content: center;
+  margin-top: 30px;
+}
+
+.product_main-photo {
+  width: 160px;
+  height: 134px;
+}
+
+.product_name {
+  font-weight: 500;
+  font-size: 12px;
+  margin-top: 45px;
+  line-height: 15px;
+}
+
+.product_code {
+  display: block;
+  margin-top: 5px;
+}
+
+.product_reviews {
+  display: flex;
+  justify-content: space-between;
+  align-content: space-between;
+  margin-top: 15px;
+  margin-bottom: 20px;
```

Don't repeat margin property
```suggestion
  margin: 15px 0 20px;
```